<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>monthPlan</class>
 <widget class="QWidget" name="monthPlan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>613</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>613</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>monthPlan</string>
  </property>
  <widget class="QListWidget" name="monthListWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>351</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>50</y>
     <width>77</width>
     <height>170</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="JanBtn">
      <property name="text">
       <string>一月</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="MarBtn">
      <property name="text">
       <string>三月</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="MayBtn">
      <property name="text">
       <string>五月</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="JulBtn">
      <property name="text">
       <string>七月</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="SepBtn">
      <property name="text">
       <string>九月</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="NovBtn">
      <property name="text">
       <string>十一月</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>50</y>
     <width>77</width>
     <height>170</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="FebBtn">
      <property name="text">
       <string>二月</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="AprBtn">
      <property name="text">
       <string>四月</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="JunBtn">
      <property name="text">
       <string>六月</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="AugBtn">
      <property name="text">
       <string>八月</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="OctBtn">
      <property name="text">
       <string>十月</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="DecBtn">
      <property name="text">
       <string>十二月</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>400</y>
     <width>355</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="numEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="planLineEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="addBtn">
      <property name="text">
       <string>新增</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
